polyssifier/polyssifier.py

```diff
@@ -185,7 +185,7 @@ def _scorer(clf, X, y):
             try:
                 ypred = ypred[:, 1]
             except:
-                print('predict proba return shape {}'.format(yprob.shape))
+                print('predict proba return shape')
```

Comment:
Did you mean to have ypred.shape orignally? yprob isn't initialized or referenced anywhere inside of the scope of _scorer so if the exception was ever caught the print statement would cause the program to crash.